src/phoenix/session/session.py

```diff
@@ -69,13 +70,15 @@ def __init__(
         reference_dataset: Optional[Dataset] = None,
         corpus_dataset: Optional[Dataset] = None,
         trace_dataset: Optional[TraceDataset] = None,
+        default_umap_parameters: Optional[Dict[str, Union[int, float]]] = None,
```

Comment:
`Any` seems fine, no need to be super precise here with the type signature. Besides, there are also string parameters such as metric names.
```suggestion
        default_umap_parameters: Optional[Mapping[str, Any]] = None,
```